### DISCUSSION TITLE
Album Name with Comma and Ellipsis Breaks After Update

### DISCUSSION THREAD

**Discussion by Andrew-McGee** [ampache/ampache#2856] [General]
I didn't raise an issue for this yet. I did a quick search and didn't find anything but thought I would check in first to see if this is known.

I have an album called "But Seriously, Folks...". Note the comma and the ellipsis - quotes added by me, not part of the album name.

Here's a screenshot before I run update - The album name displays fine and everything works.
![Overview](https://i.postimg.cc/s2nSZ6Kp/screen01.jpg)

Now after I run an update (and I think it is specifically the clean process) look what happens to the album name:
![Overview](https://i.postimg.cc/Y0Dynzpg/screen01b.jpg)

It gets double quotes added to the name and it breaks the album art. Interestingly if I go to edit the album it shows no name in the edit form. I checked in the db and the name for this album entry really has changed to include the double quotes.

When I manually edit it back without the quotes it all works fine again until next update.

Any ideas?

**Comment by Andrew-McGee** [comment]
SOLVED - it was the tags! *facepalm*

**Comment by lachlan-00** [comment]
Speaking issues out loud is a great way of tracing it down! :)
